dev_guide/templates.adoc

```diff
@@ -110,8 +109,8 @@ parameters to create a configuration with customized environment variables:
 ====
 
 ----
-$ osc process -f examples/sample-app/application-template-dockerbuild.json -v
-ADMIN_USERNAME=root,MYSQL_DATABASE=admin
+$ osc process -f examples/sample-app/application-template-dockerbuild.json \
+    -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin
```

Comment:
Instead of backslashes for multilines, I think we're preferring keeping long commands like this on a single line (an exception to the normal 80 column lines) and using `nowrap` to spawn a horizontal scroll. E.g.,:

```
====

[options="nowrap"]
----
$ osc process -f examples/sample-app/application-template-dockerbuild.json -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin
----
====
```

Live example:

http://docs.openshift.org/latest/admin_guide/master_node_configuration.html#creating-new-configuration-files